libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -446,6 +338,124 @@ impl NodeAPI for Greenlight {
         client.key_send(request).await?.into_inner().try_into()
     }
 
+    async fn start(&self) -> Result<()> {
+        self.get_client().await?;
+        Ok(())
+    }
+
+    async fn sweep(
```

Comment:
We should be consistent with naming. Currently we have `prepare_withdraw` and `sweep`.
Perhaps change it to:
`prepare_withdraw`, `withdraw`

Reply:
Agree. I'll open a separate PR changing from sweep to withdraw.